judgments/views/advanced_search.py

```diff
@@ -57,14 +58,18 @@ def _do_dates_require_warnings(
     iso_date: Optional[str], total_results: int, min_actual_year: Optional[int]
 ) -> tuple[bool, Optional[str]]:
     """
-    Check if users have requested a year before what we technically handle,
+    Check if users have requested a year before what we have available,
     if it is, then we provide a warning letting them know.
     """
-    min_year = get_minimum_valid_year()
+    min_year = get_minimum_warning_year()
 
     if not iso_date:
         return False, None
     from_date = date.fromisoformat(iso_date)
+    # If the date is 1085, then that's the default, the user hasn't typed in
+    # a non-sensical answer, don't warn about it.
+    if from_date.year == settings.MINIMUM_ALLOWED_YEAR:
```

Comment:
This makes me a bit twitchy because if a user does type "1085" then we should still warn them, but solving it is a bit of a faff.

Reply:
Yes; but it's such an edge case that I'm not really worried about it. And this fixes a much worse bug.

Reply:
I could check if the date is 1085-01-01; otherwise I'd be have to be checking if a date was provided at all